templates/report_template.html

```diff
@@ -42,6 +42,7 @@
 						<th style="text-align: center;">OSP Version </th>
 						<th style="text-align: center;">Job Name</th>
 						<th style="text-align: center;">Last Build Number</th>
+						<th style="text-align: center;">Days since last build</th>
```

Comment:
```suggestion
						<th style="text-align: center;">Days Since Last Build</th>
```
To make it consistent with the rest of the columns